contrib/build-push-ceph-container-imgs.sh

```diff
@@ -17,14 +17,34 @@ function login_docker_hub {
   docker login -u "$DOCKER_HUB_USERNAME" -p "$DOCKER_HUB_PASSWORD"
 }
 
+function create_head_or_point_release {
+  local latest_tag
+  local latest_commit_sha
+  local current_branch
+  # we test if we are running on a tagged commit
+  # if so, we build images with this particular tag
+  # otherwise we just build using the branch name (make's default)
```

Comment:
This is not true as you do override the default value with branch name & commit-id.